### ISSUE TITLE
Security for all type "except entity" if has rol only hide menu

### ISSUE THREAD

**Issue by gonzakpo** [alterphp/EasyAdminExtensionBundle#75]
It's a custom action to a route like do to hide in the menu by role:
```yaml
easy_admin:
        menu:
            - { label: 'App action', route: 'app_action', role: ROLE_SUPER_ADMIN }
```
It does not work

**Comment by alterphp** [comment]
Hi @gonzakpo. It can not work as the bundle may only control routes it handles, ie `easyadmin`. You must control access in your controller with `authorization_checker` service.

**Comment by gonzakpo** [comment]
It is visible in the menu. Proposal could develop that option and hide in the menu?

**Comment by alterphp** [comment]
We may hide the menu but not limit access to the route (someone with the URL can access it unless it is protected by security configuration or in its controller).

It's a bit tricky to understand and may lead to security issues... I don't know...

**Comment by gonzakpo** [comment]
Add documentation

**Comment by alterphp** [comment]
Implemented in v1.3.3
